src/Components/Blazor/Blazor/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -95,5 +92,24 @@ protected override Task UpdateDisplayAsync(in RenderBatch batch)
                 throw new NotImplementedException($"{nameof(WebAssemblyRenderer)} is supported only with in-process JS runtimes.");
             }
         }
+
+        /// <inheritdoc />
+        protected override bool HandleException(int componentId, IComponent component, Exception exception)
```

Comment:
We could in theory leave it unhandled in the blazor case. You get a slightly shorter stack trace if you handle it rather than have the runtime catch it.